### PR TITLE
Waf 91 update cls for waf

### PR DESCRIPTION
**Pull request by jsommerville-untangle**

### CODE REVIEW

File: services/licensemanager/LicenseManager.go [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+// @return []AppState - array of current app states
+func GetAppStates() []AppState {
+	return appStates
+}
+
+// SetAppState sets the desired state of an app
+// @param cmd AppCommand - the command to run on the app
+// @param save bool - if we should store the app state or not
+// @return error - associated errors
+func SetAppState(cmd AppCommand, save bool) error {
```
Comment (jsommerville-untangle) [inline services/licensemanager/LicenseManager.go:136]: should these be moved to AppStates or AppCommand struct files

File: services/licensemanager/AppCommand.go [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package licensemanager
+
+// Command is used for specific commands (SetState)
+type AppCommand struct {
```
Comment (jsommerville-untangle) [inline services/licensemanager/AppCommand.go:4]: rename to servicecommand, etc.

File: services/licensemanager/License.go [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+	Type        string `json:"type"`
+	End         int64  `json:"end"`
+	Start       int64  `json:"start"`
+	Seats       int64  `json:"seats" default:"-1"`
+	DisplayName string `json:"displayName"`
+	Key         string `json:"key"`
+	KeyVersion  int    `json:"keyVersion"`
+	Name        string `json:"name"`
+	JavaClass   string `json:"javaClass"`
+	Valid       bool   `json:"valid" default:"false"`
```
Comment (jsommerville-untangle) [inline services/licensemanager/License.go:15]: remove valid?

File: services/licensemanager/LicenseInfo.go [hunk 4]
```diff
@@ -0,0 +1,7 @@
+package licensemanager
+
+// LicenseInfo represents the json returned from license server
+type LicenseInfo struct {
+	JavaClass string    `json:"javaClass"`
+	List      []License `json:"list"`
```
Comment (jsommerville-untangle) [inline services/licensemanager/LicenseInfo.go:6]: Add restricted flag

**Review by rasmussen-untangle — APPROVED** [review]

**Pull request merged**